src/components/description/Description.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import { shallow, configure } from 'enzyme';
+import expect from 'expect';
+import Description from './Description';
+
+
+configure({ adapter: new Adapter() });
+
+function assertElementHasText(wrapper, query, text) {
+  expect(wrapper.find(query).text()).toEqual(text);
+}
+
+describe('Description.js', () => {
+  describe('<Description />', () => {
+    it('shows text', () => {
+      const wrapper = shallow(<Description description="Some text" />);
+      assertElementHasText(wrapper, '[className="description"]', 'Some text');
+    });
+
+    it('shows metadata', () => {
+      const layerId = 0.1234;
```

Comment:
Is this typical? Floats don't make good IDs.

Reply:
True, I have just changed it to a string, which is closer to what happens in the LayerControllerSubscriber https://github.com/hubmapconsortium/vitessce/blob/09ee411afe2a873c18f6892b1d091d7dbe270148/src/components/layer-controller/LayerControllerSubscriber.js#L37